util/cron/common-whitebox.bash

```diff
@@ -166,8 +166,13 @@ if [ "${HOSTNAME:0:6}" = "esxbld" ] ; then
 fi
 
 if [ "${COMP_TYPE}" != "HOST-TARGET-no-PrgEnv" ] ; then
-    log_info "Loading cray-fftw module."
-    module load cray-fftw
+    # We want cray-fftw with PrgEnv compilers.  But that in turns loads
+    # cray-mpich and our PGI target compiler is so old that bringing in
+    # cray-mpich has become impossible, so skip cray-fftw with PGI.
+    if [ $COMPILER != pgi ] ; then
```

Comment:
I have a minor preference for `if [ "${COMPILER}" == "pgi" ] ; then`, but it's almost certainly not worth changing